js/windowMgr.js

```diff
@@ -211,7 +211,7 @@ function doCreateMainWindow(initialUrl, initialBounds, isCustomTitleBar) {
         mainWindow.webContents.send('window-leave-full-screen');
     });
 
-    if (initialBounds && !isNodeEnv) {
```

Comment:
Rerun `full-screen.spectron.js` and `resizeWindows.spectron.js`, PASSED

Reply:
@thaisym1912 You need to reset the app state in `full-screen` and `resizeWindows` test cases because we cannot control the order of the test executions.

if `full-screen` is the first test case to run, other test cases will be affected.

Reply:
@KiranNiranjan You can see in startApplication() method, I've setSize to default and make it center().